blazor/treegrid/rows/row-template.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: Row Template in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Row Template in Syncfusion Blazor Tree Grid component and much more details.

```

Comment:
description: Check out here and learn more details about the Row Template in the Syncfusion Blazor Tree Grid component.